Jenkinsfile

```diff
@@ -202,6 +149,15 @@ VERSION      : ${env.VERSION}
             }
         } // end publish to npm
 
+        // Run PureScale when we have a new `next` version
```

Comment:
@maxwellmooney13 Zach and I talked about this, but since you missed it: Wes gave the suggestion for us to trigger the PureScale tests whenever we have a `next` version so we can get that ball rolling a little faster than we do right now.